### ISSUE TITLE
[Conversational] Property  conversational  does not exist on type  HfInference

### ISSUE THREAD

**Issue by VladSolyankin** [huggingface/huggingface.js#586]
Hi, i'm new to huggingface.js and i want to build an ai chat-bot for my pet-project. I have all other methods from docs, but i don't have hf.conversational one:
![image](https://github.com/huggingface/huggingface.js/assets/105321498/9b11d935-fc56-425b-b352-89e24102c569)
It was deleted or i'm doing something wrong? (installing via npm)

**Comment by coyotte508** [comment]
Yes this was removed in https://github.com/huggingface/huggingface.js/pull/457 in favor or `textGenerationStream`, cc @Wauplin @SBrandeis 

The docs should be updated indeed

**Comment by VladSolyankin** [comment]
Ok, thank you!

**Comment by coyotte508** [comment]
(Reopening, we should fix the doc!)
